diff -up pimcommon-20.03.90/src/pimcommonakonadi/collectionpage/incidencesforwidget.cpp.omv~ pimcommon-20.03.90/src/pimcommonakonadi/collectionpage/incidencesforwidget.cpp
--- pimcommon-20.03.90/src/pimcommonakonadi/collectionpage/incidencesforwidget.cpp.omv~	2020-04-09 22:42:00.092807693 +0200
+++ pimcommon-20.03.90/src/pimcommonakonadi/collectionpage/incidencesforwidget.cpp	2020-04-09 22:42:06.926889361 +0200
@@ -69,7 +69,7 @@ IncidencesForWidget::IncidencesForWidget
 
     d->mIncidencesForComboBox->setObjectName(QStringLiteral("contentstypecombobox"));
     d->mIncidencesForComboBox->setWhatsThis(whatsThisForMyOwnFolders);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(d->mIncidencesForComboBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &IncidencesForWidget::currentIndexChanged);
 #else
     connect(d->mIncidencesForComboBox, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &IncidencesForWidget::currentIndexChanged);
